lib/ansible/modules/known_hosts.py

```diff
@@ -101,6 +101,8 @@ def enforce_state(module, params):
     host = params["name"].lower()
     key = params.get("key", None)
     path = params.get("path")
+    if not os.path.exists(path):
+        module.fail_json(msg="Failed to find file %s" % path)
```

Comment:
This check appears unnecessary, since the `stat` call below will also provide a user-friendly error message if the file does not exist.